docs/crds/traces.gadget.kinvolk.io.md

```diff
@@ -64,7 +64,7 @@ spec:
   node: minikube
   gadget: dns
   filter:
-    namespace: kube-system
+    namespace: gadget
     podname: etcd-minikube
```

Comment:
I think this namespace should not be changed since it is about the etcd pod.

Same thing in other files:
- docs/gadgets/dns.md
- pkg/resources/samples/gadget.kinvolk.io_v1alpha1_trace.yaml
- pkg/resources/samples/trace-dns.yaml

Reply:
I did this modification otherwise the CI would cry:
https://github.com/kinvolk/inspektor-gadget/runs/4462010871?check_suite_focus=true#step:10:41

Reply:
The CI error is because the generated file (markdown files) is not in sync with the source (yaml files). If you edit the source (yaml files), then run `make generate-documentation` to generate the markdown files, and commit both, then the CI should pass.

Reply:
Oh OK! I will just remove the `*.md` from the commit!